packages/css/src/paragraph/paragraph.scss

```diff
@@ -5,28 +5,45 @@
 
 @import "../../utils/calculate-fluid-style";
 
+@import "../../node_modules/@amsterdam/design-system-tokens/dist/variables";
+
 @mixin reset {
   box-sizing: border-box;
   -webkit-text-size-adjust: 100%;
 }
 
 .amsterdam-paragraph {
-  color: var(--amsterdam-color-primary-black);
-  font-family: var(--amsterdam-typography-font-family);
-  font-size: calculate-fluid-style(18, 24);
-  font-weight: var(--amsterdam-typography-font-weight-normal);
-  line-height: calculate-fluid-style(29, 38);
+  color: var(--amsterdam-paragraph-color);
+  font-family: var(--amsterdam-paragraph-font-family);
+
+  /*
+    We don't use the amsterdam-paragraph-font-size variable, because we generate a somewhat
+    complex css rule with a Sass function. The variable is added here so people can
+    overwrite that css rule with their own value.
+  */
+  font-size: var(
+    --amsterdam-paragraph-font-size,
+    calculate-fluid-style($amsterdam-typography-font-size-18, $amsterdam-typography-font-size-24)
+  );
+  font-weight: var(--amsterdam-paragraph-font-weight);
+  line-height: var(--amsterdam-paragraph-line-height);
   margin-block: 0;
```

Comment:
Should this be moved into the reset mixin?

Reply:
Hm, I'm not sure. A reset usually fixes differences between browsers. (Almost?) all browsers set `margin-top` and `margin-bottom` for paragraphs, so it's not different between browsers.

Reply:
We decided that we’re mainly interested in resolving the gap between browsers and our components, and less in whether something’s a ‘reset’ or a ‘normalise’. We’ll include both in a mixin called `reset`.